core/src/main/java/tc/oc/pgm/timelimit/TimeLimitModule.java

```diff
@@ -83,14 +85,25 @@ public TimeLimitModule parse(MapFactory factory, Logger logger, Document doc)
       return oldTimeLimit;
     }
 
-    private static @Nullable TimeLimit parseTimeLimit(MapFactory factory, Element el)
+    private static @Nullable TimeLimit parseTimeLimit(MapFactory factory, Element el, String child)
         throws InvalidXMLException {
-      el = el.getChild("time");
+      el = el.getChild(child);
       if (el == null) return null;
 
+      return parseTimeLimit(factory, el);
+    }
+
+    public static TimeLimit parseTimeLimit(MapFactory factory, Element el)
+        throws InvalidXMLException {
+      String durationText = el.getTextNormalize();
+      Duration duration =
+          durationText == null || durationText.isEmpty()
+              ? XMLUtils.parseDuration(el.getAttribute("duration"))
```

Comment:
I'd say let's not modify the syntax and keep it as 
`<timelimit>5m</timelimit>` instead of `duration="5m"`

at the end of the day it's the timelimit syntax